ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCliBase.scala

```diff
@@ -354,6 +354,12 @@ class CommonCliBase(name: LedgerName) {
           "Maximum command deduplication duration."
         )
 
+      opt[Unit]("use-error-codes-v2")
+        .optional()
+        .hidden()
+        .text("Enable new self-service errors.")
```

Comment:
I'd strongly discourage using any time references here. Maybe rewrite it to `Enable self-service error codes.`?